assets/js/temp3.js

```diff
@@ -0,0 +1,226 @@
+// Global Variables
+
+var state = {
+  // starting state of questions in DOM - none.
+  items: [],
+  // answers[] is static but will be changed out with every question iteration
+  answers: [
+    '0815',
+    '2B',
+    'BAM128',
+    'Barely'
+  ],
+  // starting value of question when DOM Loads.
+  current_question: 0,
+  // starting value shown as score in DOM.
+  correct: 0,
+  // starting value of counter
+  index: 0,
+  // sets questionHTML to be 'this.'
+  questionHTML: null
+};
+
+var QUESTIONS = [
```

Comment:
I would conform to lower case or camel case for object names, even with larger structures.